packages/flutter_tools/test/general.shard/base/command_help_test.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/command_help.dart';
+import 'package:flutter_tools/src/base/terminal.dart' show OutputPreferences, outputPreferences;
+import 'package:mockito/mockito.dart';
+import 'package:platform/platform.dart';
+
+import '../../commands.shard/hermetic/build_fuchsia_test.dart';
```

Comment:
This is the problematic import

Reply:
Oh wow. I totally missed that. I should not relay on the IDE so much ...